<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditArguments</class>
 <widget class="QDialog" name="EditArguments">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Arguments</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="argList"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addArgButton">
       <property name="text">
        <string>Add Argument</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addPathButton">
       <property name="text">
        <string>Add Path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editArgButton">
       <property name="text">
        <string>Edit Argument</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeArgButton">
       <property name="text">
        <string>Remove Argument</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moveUpArgButton">
       <property name="text">
        <string>Move Up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moveDownArgButton">
       <property name="text">
        <string>Move Down</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditArguments</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditArguments</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
